jax/_src/numpy/fft.py

```diff
@@ -522,18 +522,131 @@ def irfft(a: ArrayLike, n: int | None = None,
   return _fft_core_1d('irfft', xla_client.FftType.IRFFT, a, n=n, axis=axis,
                       norm=norm)
 
-@implements(np.fft.hfft)
+
 def hfft(a: ArrayLike, n: int | None = None,
          axis: int = -1, norm: str | None = None) -> Array:
+  """Compute a 1-D FFT of an array whose spectrum has Hermitian symmetry.
+
+  JAX implementation of :func:`numpy.fft.hfft`.
+
+  Args:
+    a: input array.
+    n: optional, int. Specifies the dimension of the result along ``axis``. If
+      not specified, ``n = 2*(m-1)``, where ``m`` is the dimension of ``a``
+      along ``axis``.
+    axis: optional, int, default=-1. Specifies the axis along which the transform
+      is computed. If not specified, the transform is computed along axis -1.
+    norm: optional, string. The normalization mode. "backward", "ortho" and "forward"
+      are supported. Default is "backward".
+
+  Returns:
+    An array containing the one-dimensional discret Fourier transform of ``a``
+    by exploiting its inherent Hermitian-symmetry, having a dimension of ``n``
+    along ``axis``.
```

Comment:
Is it worth stating that the output will always be real?

Reply:
Written it as "A real-vlaued array"